kornia/geometry/calibration/distort.py

```diff
@@ -96,9 +105,14 @@ def distort_points(points: torch.Tensor, K: torch.Tensor, dist: torch.Tensor) ->
     cy: torch.Tensor = K[..., 1:2, 2]  # princial point in y (Bx1)
     fx: torch.Tensor = K[..., 0:1, 0]  # focal in x (Bx1)
     fy: torch.Tensor = K[..., 1:2, 1]  # focal in y (Bx1)
+    new_cx: torch.Tensor = new_K[..., 0:1, 2]  # princial point in x (Bx1)
```

Comment:
cx is never used then ?

Reply:
used in line 148

Reply:
moved def of vars to the place where they are used